test/tests/functional/pbs_resource_usage_log.py

```diff
@@ -300,7 +300,7 @@ def test_acclog_force_requeue(self):
         self.mom.signal('-KILL')
 
         # Verify that nodes are reported to be down.
-        self.server.expect(NODE, {ATTR_NODE_state: 'down'},
+        self.server.expect(NODE, {ATTR_NODE_state: (MATCH_RE, 'down')},
```

Comment:
I don't think it's necessary (because we use `re.search` instead of `re.match`), but I would use `(MATCH_RE, '.*down.*')`.

Reply:
Yes. MATCH_RE does a re.search.  So I feel (MATCH_RE, 'down') will work fine. I will keep it as is.